perf_test/blas/blas/KokkosBlas_trtri_perf_test.hpp

```diff
@@ -0,0 +1,424 @@
+}
+
+template <class ExecutionSpace>
+struct parallel_blas_trtri {
+  trtri_args_t trtri_args_;
+
+  parallel_blas_trtri(trtri_args_t trtri_args) : trtri_args_(trtri_args) {}
+
+  KOKKOS_INLINE_FUNCTION
+  void operator()(const int& i) const {
+    auto svA = Kokkos::subview(trtri_args_.A, i, Kokkos::ALL(), Kokkos::ALL());
+
+    KokkosBlas::trtri(&trtri_args_.uplo, &trtri_args_.diag, svA);
+  }
+};
+
+template <class scalar_type, class vta, class device_type>
+void __do_trtri_parallel_blas(options_t options, trtri_args_t trtri_args) {
+  uint32_t warm_up_n = options.warm_up_n;
+  uint32_t n         = options.n;
+  Kokkos::Timer timer;
+  using execution_space = typename device_type::execution_space;
+  using functor_type    = parallel_blas_trtri<execution_space>;
+  functor_type parallel_blas_trtri_functor(trtri_args);
+
+  STATUS;
+
+  Kokkos::parallel_for("parallelBlasWarmUpLoopTrtri",
+                       Kokkos::RangePolicy<execution_space>(0, warm_up_n),
+                       parallel_blas_trtri_functor);
+  Kokkos::fence();
+
+  timer.reset();
+  Kokkos::parallel_for("parallelBlasTimedLoopTrtri",
+                       Kokkos::RangePolicy<execution_space>(0, n),
+                       parallel_blas_trtri_functor);
+  Kokkos::fence();
+  __trtri_output_csv_row(options, trtri_args, timer.seconds());
+  return;
+}
+
+template <class uplo, class diag, class tag, class ExecutionSpace>
+struct parallel_batched_trtri {
+  trtri_args_t trtri_args_;
+
+  parallel_batched_trtri(trtri_args_t trtri_args) : trtri_args_(trtri_args) {}
+
+  KOKKOS_INLINE_FUNCTION
+  void operator()(const int& i) const {
+    auto svA = Kokkos::subview(trtri_args_.A, i, Kokkos::ALL(), Kokkos::ALL());
+
+    SerialTrtri<uplo, diag, tag>::invoke(svA);
+  }
+};
+
+template <class uplo, class diag, class device_type>
+void __do_trtri_parallel_batched_template(options_t options,
+                                          trtri_args_t trtri_args) {
+  uint32_t warm_up_n = options.warm_up_n;
+  uint32_t n         = options.n;
+  Kokkos::Timer timer;
+  using tag             = Algo::Trtri::Unblocked;
+  using execution_space = typename device_type::execution_space;
+  using functor_type = parallel_batched_trtri<uplo, diag, tag, execution_space>;
+  functor_type parallel_batched_trtri_functor(trtri_args);
+
+  STATUS;
+
+  Kokkos::parallel_for("parallelBatchedWarmUpLoopTrtri",
+                       Kokkos::RangePolicy<execution_space>(0, warm_up_n),
+                       parallel_batched_trtri_functor);
+  Kokkos::fence();
+
+  timer.reset();
+  Kokkos::parallel_for("parallelBatchedTimedLoopTrtri",
+                       Kokkos::RangePolicy<execution_space>(0, n),
+                       parallel_batched_trtri_functor);
+  Kokkos::fence();
+  __trtri_output_csv_row(options, trtri_args, timer.seconds());
+
+  return;
+}
+
+template <class scalar_type, class vta, class device_type>
+void __do_trtri_parallel_batched(options_t options, trtri_args_t trtri_args) {
```

Comment:
@e10harvey I just have one comment: should we place ```batched``` tests in ```perf_test/batched``` instead of in here?

Reply:
@vqd8a: Thanks for you feedback! I'm OK with leaving the batched tests here since both the TRMM and TRTRI blas fall back routines invoke the serial batched routines internally.

Reply:
@e10harvey Thanks. That is fine to me.